claat/cmd/export.go

```diff
@@ -169,6 +169,16 @@ func writeCodelabWriter(w io.Writer, clab *types.Codelab, extraVars map[string]s
 		Steps:    clab.Steps,
 		Extra:    extraVars,
 	}}
+
+	if ctx.Format != "offline" {
+		ext := ctx.Format
```

Comment:
Does this end up getting used somewhere? Do we need the intermediate variable?

Reply:
I actually think we should remove the "offline" support for the Memory Export because  offline should require multiple io.Writer instead of one to make it work properly.